packages/create/src/build.config.ts

```diff
@@ -35,7 +41,27 @@ export async function before({ project }: Project.Context) {
             if (!entrypoint && filename.indexOf("/") === -1) {
                 entrypoint = filename;
             }
-            await cp(file, createPkg.resolve("dist/templates/", name, filename));
+            const source = await readFile(file, "utf-8");
+
+            for (const [, pkgName] of source.matchAll(/from "(@[^/"]+\/[^/"]+|[^/"]+)[^"]*"/g)) {
+                if (dependencies[pkgName]) {
+                    continue;
+                }
+
+                if (pkgName.startsWith("@matter/")) {
```

Comment:
should we also handle the @project-chip/matter.js dep here and add?

Reply:
yes, fixed.  It worked before but only because the ble package (incorrectly) had a dependency on @project-chip/matter.js